Makefile.core.mk

```
@@ -284,7 +284,8 @@ refresh-goldens:
 		./pkg/bootstrap/... \
 		./pkg/kube/inject/... \
 		./pilot/pkg/security/authz/builder/... \
-		./cni/pkg/plugin/...
+		./cni/pkg/plugin/... \
+		./istioctl/cmd/...
```

```suggestion
		./istioctl/cmd/...
```

Why is this added here? 

(I confess I don't understand what 'goldens' are - I presume based on comments they're binaries that rely on codegen'd sources in which case this addition makes sense/I don't understand why it wasn't already there)